libs/npm/core/src/components/navbar/NavBarGroup.tsx

```diff
@@ -2,14 +2,20 @@ import React from 'react';
 
 import * as styled from './styled';
 
+export interface INavBarGroupProps extends React.HTMLAttributes<HTMLDivElement> {
+  /** whether this part of the navbar should hover over the screen with a higher z-index */
+  hover?: boolean;
+}
+
 /**
  * The element that groups the various navigation bar items together.
  * @param children the navigation bar items
  * @returns navigation bar group
  */
-export const NavBarGroup: React.FC<React.HTMLAttributes<HTMLDivElement>> = ({
-  children,
-  ...rest
-}) => {
-  return <styled.NavBarGroup {...rest}>{children}</styled.NavBarGroup>;
+export const NavBarGroup: React.FC<INavBarGroupProps> = ({ children, hover, ...rest }) => {
```

Comment:
You don't need to extract `hover` if you're just going to place it on the same line as `{...rest}`